Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py

```diff
@@ -1286,20 +1281,23 @@ def fetch_incidents():
                     incidents.append(incident)
 
             if len(incidents) == INCIDENTS_PER_FETCH:
-                demisto.setLastRun({'first_behavior_detection_time': prev_fetch,
-                                    'detection_offset': offset + INCIDENTS_PER_FETCH})
+                demisto_set_last_run['first_behavior_detection_time'] = prev_fetch
+                demisto_set_last_run['detection_offset'] = offset + INCIDENTS_PER_FETCH
             else:
-                demisto.setLastRun({'first_behavior_detection_time': last_fetch_time})
+                demisto_set_last_run['first_behavior_detection_time'] = last_fetch_time
 
     if 'Incidents' in fetch_incidents_or_detections:
         incident_type = 'incident'
 
         last_fetch_time, offset, prev_fetch, last_fetch_timestamp = get_fetch_times_and_offset(incident_type)
+        last_run = demisto.getLastRun()
+        last_incident_fetch = last_run.get('last_incident_fetch')
+        last_new_incident_fetched = ''
```

Comment:
```suggestion
        new_last_incident_fetched = ''
```

change in all subsequent locations

Reply:
done